src/public/meeting.ts

```diff
@@ -143,4 +154,73 @@ export namespace meeting {
+   * liveStreamState can either contain a LiveStreamState value, or null when operation fails
+   */
+  export function getLiveStreamState(
+    callback: (error: SdkError | null, liveStreamState: LiveStreamState | null) => void,
+  ): void {
+    if (!callback) {
+      throw new Error('[get live stream state] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('meeting.getLiveStreamState');
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * Allows an app to request the live streaming be started at the given streaming url
+   * @param streamUrl the url to the stream resource
+   * @param streamKey the key to the stream resource
+   * @param callback Callback contains 2 parameters: error and liveStreamState.
+   * error can either contain an error of type SdkError, in case of an error, or null when operation is successful
+   * liveStreamState can either contain a LiveStreamState value, or null when operation fails
+   */
+  export function requestStartLiveStreaming(
+    callback: (error: SdkError | null, liveStreamState: LiveStreamState | null) => void,
+    streamUrl: string,
+    streamKey?: string,
+  ): void {
+    if (!callback) {
+      throw new Error('[request start live streaming] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('meeting.requestStartLiveStreaming', [streamUrl, streamKey]);
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * Allows an app to request the live streaming be stopped at the given streaming url
+   * @param callback Callback contains 2 parameters: error and liveStreamState.
+   * error can either contain an error of type SdkError, in case of an error, or null when operation is successful
+   * liveStreamState can either contain a LiveStreamState value, or null when operation fails
+   */
+  export function requestStopLiveStreaming(
+    callback: (error: SdkError | null, liveStreamState: LiveStreamState | null) => void,
+  ): void {
+    if (!callback) {
+      throw new Error('[request stop live streaming] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('meeting.requestStopLiveStreaming');
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * Allows an app to register an event handler to get streaming state updates
+   * @param handler Handler contains 1 parameter: liveStreamState
+   * liveStreamState will contain a LiveStreamState value
+   */
+  export function registerLiveStreamChangedHandler(handler: (liveStreamState: LiveStreamState) => void): void {
+    if (!handler) {
+      throw new Error('[register live stream changed handler] Handler cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('meeting.registerLiveStreamChangedHandler');
```

Comment:
This will not achieve the intended use. Use examples from `private/remoteCamera.ts` as a guide to creating this functionality.

Reply:
fixed